packages/map/examples/src/map/pages/demo/demo.page.html

```diff
@@ -25,7 +25,7 @@ <h1 class="h3 u-margin-bottom">Map</h1>
 			<aui-leaflet-fullscreen-control></aui-leaflet-fullscreen-control>
 		</div>
 		<div controls bottom left>
-			<aui-leaflet-locate-control></aui-leaflet-locate-control>
+			<aui-leaflet-locate-control zoomlevel=16></aui-leaflet-locate-control>
```

Comment:
Thanks for updating the example, but don't forget to update the code example in the `demo.page.ts` file.

Reply:
Please add double quotes around your prop.